app/livechat/client/views/app/tabbar/chatRoomHistoryItem.js

```diff
@@ -0,0 +1,31 @@
+import moment from 'moment';
+import './chatRoomHistoryItem.html';
+import { Template } from 'meteor/templating';
+import { ReactiveVar } from 'meteor/reactive-var';
+
+Template.chatRoomHistoryItem.helpers({
+	closedAt() {
+		const { closedAt } = Template.instance().room.get();
+		return moment(closedAt).format('LT');
+	},
+	hasClosingRoomMessage() {
+		const { lastMessage } = Template.instance().room.get();
+		if (lastMessage.t === 'livechat-close') {
```

Comment:
This is not working and it's not the right way to implement it.
You need to fetch the `livechat-close` from the database and set this to the `closingRoomMessage` ReactiveVar.

Then, you can check here if the `closingRoomMessage` exists or not.